src/views/Services.md

```diff
@@ -0,0 +1,51 @@
+<div class="container-fluid monarch-view services">
+    <h2 class="page-title">Monarch Web Services</h2> 
+    <div class="row architecture">
```

Comment:
This div is having issues with overlap in full resolution on laptop (it may or may not have something to do  with the modifications to accommodate large screens ([file here](https://github.com/monarch-initiative/monarch-ui/pull/162/commits/9c78fe96d05db200dc76ff915026f8ac8c0b4175#diff-ea994a51b0d9884f4fc046eca11a5c99)). 

Working on a 2560 x 1600 Retina display on Mac, I get this error (columns mounted on top of each other) in Chrome, Safari, and Firefox - at 100% zoom on screen. If I reduce zoom to 80% or less, then the two columns look separated. 
<img width="1417" alt="Screen Shot 2019-08-07 at 10 37 36 AM" src="https://user-images.githubusercontent.com/1714621/62644945-272e6000-b900-11e9-911f-3504ec864d33.png">

Reply:
still getting use to the new bootstrap columns, seems like they haven't solved the giant resolutions yet.

Reply:
@iimpulse With clean cache, I still see the same error. 
<img width="457" alt="Screen Shot 2019-08-07 at 1 56 58 PM" src="https://user-images.githubusercontent.com/1714621/62657422-4f778800-b91b-11e9-95d4-c72fa3d80b77.png">